apis/system-settings.md

```diff
@@ -0,0 +1,58 @@
+---
+description: Deep linking to System Settings
+---
+
+# System Settings
+
+Some of the features of your application may depend on system level settings or services, such as Internet connectivity or app level access to the network. While your app cannot directly change those settings, it can prompt users to change them by linking to specific panes in the System Settings.
+
+elementary OS implements a special `settings://` URI scheme for linking to System Settings. The links lead to specific settings panes. You can find most commonly used ones in the `Granite.SettingsUri` namespace in the Granite library.
+
+## Opening a link
+
+First, make sure you've included Granite in the build dependencies declared in your `meson.build` file:
+
+{% code title="meson.build" %}
+```coffeescript
+executable(
+    meson.project_name(),
+    'src' / 'Application.vala',
+    dependencies: [
+        dependency('granite-7'),
+        dependency('gtk4')
```

Comment:
Not sure if we wanna add versions here since this requires the latest platform
```suggestion
        dependency('granite-7', version: '>= 7.3.0'),
        dependency('gtk4', version: '>= 4.10.0')
```

Reply:
Good idea! I called it out in the copy itself too.